src/framework/ui/modal/modal.component.tsx

```diff
@@ -66,33 +66,123 @@ export type ModalElement = React.ReactElement<ModalProps>;
  *
  * @property ViewProps
  *
- * @example Modal usage and API example
+ * @overview-example Modal simple usage and API example
```

Comment:
Let's replace any `and API` examples with a simpler one demonstrating a minimum code that will make the component work